lib/internal/tls/secure-context.js

```diff
@@ -161,11 +161,9 @@ function configSecureContext(context, options = {}, name = 'options') {
       for (let i = 0; i < key.length; ++i) {
         const val = key[i];
         const pem = (
-          val !== undefined && val !== null &&
-          val.pem !== undefined ? val.pem : val);
+          val?.pem !== undefined ? val.pem : val);
```

Comment:
```suggestion
          val?.pem ?? val);
```

Reply:
What if `val.pem` is `null`?

Reply:
Using `??` would change the current behavior. If we want to keep this a refactoring, we can't use `??`. Now, if we know that `val.pem` can never be `null`, we can make that change safely - but maybe in a separate commit?